app/View/Elements/genericElements/IndexTable/headers.ctp

```diff
@@ -31,5 +31,8 @@
             __('Actions')
         );
     }
-    echo $headersHtml;
+    $thead = '<thead>';
```

Comment:
For next time if possible: Have generic changes like these in a separate commit, makes reviewing easier (don't change it for now though!)

Reply:
Sure! The problem was that I fixed the indexTable in this branch but in the meanwhile, it was also fixed in 2.4..
Sorry for that ;)